game/addons/keychain/Keychain.gd

```diff
@@ -22,9 +24,12 @@ var ignore_ui_actions := true
 ## and the fourth for [InputEventJoypadMotion]s.
 var changeable_types: PackedByteArray = [true, true, true, true]
 ## The file path of the [code]config_file[/code].
-var config_path := "user://cache.ini"
+var config_path := "user://config.ini"
 ## Used to store the settings to the filesystem.
 var config_file: ConfigFile
+## Used to check if unused binding check should be ignored for action
+var keep_binding_check : Callable = func(action_name : StringName) -> bool:
+	return action_name.begins_with("button_") and action_name.ends_with("_hotkey")
```

Comment:
Why is this a `Callable` rather than a function (even a `static` function)?

Reply:
Because I'm trying to consider a way to remove reliance on hardcoding the button hotkey check from within Keychain, I want to minimize modifications to it.